cibuildwheel/macos.py

```diff
@@ -246,14 +248,14 @@ def build(options: BuildOptions) -> None:
             # and not the repo code)
             test_command_prepared = prepare_command(
                 options.test_command,
-                project=os.path.abspath('.'),
-                package=os.path.abspath(options.package_dir)
+                project=Path('.').resolve(),
+                package=options.package_dir.resolve()
             )
             call(test_command_prepared, cwd=os.environ['HOME'], env=virtualenv_env, shell=True)
 
             # clean up
             shutil.rmtree(venv_dir)
 
         # we're all done here; move it to output (overwrite existing)
-        dst = os.path.join(options.output_dir, os.path.basename(repaired_wheel))
-        shutil.move(repaired_wheel, dst)
+        dst = options.output_dir / repaired_wheel.name
+        shutil.move(str(repaired_wheel), dst)
```

Comment:
Mayybe this `str()` can also go?

Reply:
Yes, I just tested it, and that should work, but mypy was complaining here. I don't really know why :-(
The other alternative is shutting up mypy?